src/Microsoft.DotNet.Interactive.Tests/LanguageKernelTests.cs

```diff
@@ -31,6 +31,45 @@ public LanguageKernelTests(ITestOutputHelper output) : base(output)
         {
         }
 
+        [Theory]
+        [InlineData(Language.FSharp)]
+        [InlineData(Language.CSharp)]
+        public async Task it_fails_to_get_value_with_unsupported_mimetype(Language language)
+        {
+            var valueName = "x";
+            var mimeType = "unsupported-mimeType";
+
+            var kernel = CreateKernel(language);
+
+            var source = language switch
+            {
+                Language.FSharp => new[]
+                {
+                    $"let {valueName} = 123"
+                },
+
+                Language.CSharp => new[]
+                {
+                    $"var {valueName} = 123;"
+                }
+            };
+
+            await SubmitCode(kernel, source);
+
+            await kernel.SendAsync(new RequestValue(valueName, kernel.Name, mimeType));
+
+            KernelEvents
+                .OfType<CommandFailed>()
+                .Last()
```

Comment:
Let's avoid using LINQ in assertions. Shouldn't this be `KernelEvents.Should().ContainSingle<CommandFailed>()...`?

Reply:
indeed